code/game/objects/items/food/meat.dm

```diff
@@ -179,6 +179,26 @@
 	foodtypes = SEAFOOD | VEGETABLES
 	w_class = WEIGHT_CLASS_SMALL
 
+/obj/item/food/meat_poke
+	name = "meat poke"
+	desc = "Simple poke, rice on the bottom, vegetables and meat on top. Should be mixed before eating."
+	icon = 'icons/obj/food/soupsalad.dmi'
+	icon_state = "pokemeat"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/nutriment/vitamin = 4, /datum/reagent/consumable/nutriment/protein = 2)
+	tastes = list("rice and meat" = 4, "lettuce" = 2, "soy sauce" = 2)
+	foodtypes = SEAFOOD | VEGETABLES
```

Comment:
Did you mean to make this `MEAT | VEGETABLES`?

Reply:
maybe `SEAFOOD | MEAT | VEGETABLES`?
if seaweed is vegetable not seafood, i'll change that

Reply:
I was thinking more for the fish, but yeah `SEAFOOD | MEAT | VEGETABLES` would be better since it has the meat.